roles/matrix-jitsi/defaults/main.yml

```diff
@@ -12,6 +12,9 @@ matrix_jitsi_enable_av_moderation: true
 # Authentication type, must be one of internal, jwt or ldap. Currently only
 # internal and ldap are supported by this playbook.
 matrix_jitsi_auth_type: internal
+matrix_jitsi_prosody_auth_internal_accounts:
+  - username: "jitsti-moderator"
+    password: ""
```

Comment:
This was my train of thought on the matter:

I feel like we may indeed want to enforce internal user configuration through vars.yml, so we don't end up with user confusion when migrating the playbook (regarding which users are automatically configured, and which were created manually).

By default, this will fail if a password isn't configured. My original idea was to perhaps include it in "jitsi-generate-passwords.sh" or simply allow it to fail if a user hasn't configured at least one password. Otherwise, leave it entirely blank and the behavior will be the same.

Reply:
Please let me know what solution you feel is best, and I'll go ahead and make those changes :)

Reply:
Just realized I had forgotten to commit the config validation, which is what I referred to by the default behavior. I think it's a reasonable way to handle it, but do please let me know what solution you feel is best, and I'll go ahead and make those changes :)

Reply:
Regarding account cleanup/removal, the only way I'm thinking of to accomplish this would be to do one of the following:

- Create & delete accounts on service start/stop, which I feel is a terrible idea as I imagine the account details would be stored in the .service file or otherwise on the server.

- Delete all user accounts prior to creating them, which would also enforce user configuration through vars.yml when using internal authentication. I feel this is preferable, to keep user configuration under a single source of truth and avoid user error.

As I mentioned, I do believe we should sanitize the list. To do so, we'd download and include the "mod_listusers" Prosody module so that we can iterate through the list of existing users with "prosodyctl deluser".

mod_listusers: http://prosody.im/files/mod_listusers.lua

If you like this solution, I'm happy to give implementing it a shot.